.all-contributorsrc

```diff
@@ -37,7 +37,20 @@
       "contributions": [
         "code",
         "doc",
-        "test"
+        "test",
```

Comment:
Do you mean unit tests, or something else? This is the only contributor with 'test'

Reply:
Any type of written tests. I have updated the three primary contributors to make it consistent.